bundles/org.openhab.binding.warmup/src/main/java/org/openhab/binding/warmup/internal/api/MyWarmupConfigurationDTO.java

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.warmup.internal.api;
+
+/**
+ * The {@link MyWarmupConfigurationDTO} class contains fields mapping thing configuration parameters.
+ *
+ * @author James Melville - Initial contribution
+ */
+public class MyWarmupConfigurationDTO implements Comparable<MyWarmupConfigurationDTO> {
+
+    public String username;
+    public String password;
+
+    @Override
+    public int compareTo(MyWarmupConfigurationDTO o) {
```

Comment:
I'd initially put this in so that the existing session token was not discarded if the username/pw in the configuration were unchanged when the bridge was reconfigured. I've now taken it out because it's probably unnecessary complexity and I've not seen the API get unhappy with frequent logins.